legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/sqlQueryToString/SQLQueryToString.pure

```diff
@@ -1222,22 +1237,38 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::getDy
                                                  choice(DatabaseType.Presto,       $allStates,            ^ToSql(format=regexpLikePattern('%s','%s'), transform={p:String[2]|$p->transformRegexpParams($dbType)}))]),
       forDynafunction('max',                   [ choice($allTypes,                 $allStates,            ^ToSql(format='max(%s)'))]),
       forDynafunction('min',                   [ choice($allTypes,                 $allStates,            ^ToSql(format='min(%s)'))]),
-      forDynafunction('minute',                [ choice($allTypes,                 $allStates,            ^ToSql(format='minute(%s)'))]),
+     forDynafunction('minute',                [ choice($aSybase,                  $allStates,            ^ToSql(format='minute(%s)')),

```

Comment:
It seems this is removing support for some databaseTypes - Can we please confirm all db types are supported after the change too?